.gitignore

```diff
@@ -18,3 +18,4 @@ npm-debug.log
 /public/css
 /public/fonts
 /public/js
+/resources/fonts/dompdf_font_family_cache.php
```

Comment:
Wasn't this in another PR also?

Reply:
yes,i suspect it got whacked by another branch.